react/features/base/react/components/web/Watermarks.js

```diff
@@ -173,18 +210,29 @@ class Watermarks extends Component<Props, State> {
      */
     _renderJitsiWatermark() {
         let reactElement = null;
-
-        if (this.state.showJitsiWatermark
-                || (this.props._isGuest
-                    && this.state.showJitsiWatermarkForGuests)) {
-            reactElement = <div className = 'watermark leftwatermark' />;
-
-            const { jitsiWatermarkLink } = this.state;
-
-            if (jitsiWatermarkLink) {
+        const {
+            _customLogoUrl,
+            _customLogoLink
+        } = this.props;
+
+        if (this._canDisplayJitsiWatermark()) {
+            const link = _customLogoLink || this.state.jitsiWatermarkLink;
+            const style = _customLogoUrl ? {
+                backgroundImage: `url(${_customLogoUrl})`,
```

Comment:
```suggestion
                backgroundImage: `url(${_customLogoUr || interfaceConfig.DEFAULT_LOGO_URL})`,
```

Why don't we do this so the style is applied also for the default logo? we can remove the conditional.